website/docs/components/app-footer/partials/code/component-api.md

```diff
@@ -0,0 +1,121 @@
+    Set the overall theme use by the component and its children.
+  </C.Property>
+  <C.Property @name="ariaLabel" @type="string" @default="&quot;Footer items&quot;">
+    Accepts a localized string. The `ariaLabel` value is applied to the list of `AppFooter` content.
+  </C.Property>
+  <C.Property @name="copyrightYear" @type="string" @default="currentYear">
+    Pass a custom year value for the `Copyright` instead of the default current year value.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### AppFooter::LegalLinks
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="hrefForSupport" @type="string" @default="&quot;https://www.hashicorp.com/support&quot;">
+    Override the default href value with a custom url value.
+  </C.Property>
+  <C.Property @name="hrefForTerms" @type="string" @default="&quot;https://www.hashicorp.com/terms-of-service&quot;">
+    Override the default href value with a custom url value.
+  </C.Property>
+  <C.Property @name="hrefForPrivacy" @type="string" @default="&quot;https://www.hashicorp.com/privacy&quot;">
+    Override the default href value with a custom url value.
+  </C.Property>
+  <C.Property @name="hrefForSecurity" @type="string" @default="&quot;https://www.hashicorp.com/security&quot;">
+    Override the default href value with a custom url value.
+  </C.Property>
+  <C.Property @name="Accessibility" @type="string" @default="&quot;mailto:accessibility@hashicorp.com&quot;">
+    Override the default href value with a custom url value.
+  </C.Property>
+  <C.Property @name="ariaLabel" @type="string" @default="&quot;Legal links&quot;">
+    Accepts a localized string. The `ariaLabel` value is applied to the nested list of included legal links.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### AppFooter::StatusLink
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="status" @type="string" @values={{array "operational" "degraded" "maintenance" "critical" }}>
+    Passing one of the defined values sets the associated `text`, `statusIcon`, and `statusIconColor`. Either `status` or `text` must be passed or an error will be thrown.
+  </C.Property>
+  <C.Property @name="statusIcon" @type="string">
+    Pass a custom icon name.
+  </C.Property>
+  <C.Property @name="statusIconColor" @type="string">
+    Pass a custom icon color. Accepts any valid CSS color value.
+  </C.Property>
+  <C.Property @name="text" @type="string">
+    Pass a custom text value.
+  </C.Property>
+  <C.Property @name="href" @type="string" @default="https://status.hashicorp.com">
```

Comment:
The status link supports not just the `@href` but also the `@route` arguments, but is very unlikely they will be used. So I see why not document it here. Was this the thinking?

Reply:
Yes but I can add all that.

Reply:
Pushed up